pkg/logentry/stages/timestamp_test.go

```diff
@@ -455,9 +475,17 @@ func TestTimestampStage_ProcessActionOnFailure(t *testing.T) {
 				extracted := inputEntry.extracted
 				timestamp := inputEntry.timestamp
 				entry := ""
-
-				s.Process(inputEntry.labels, extracted, &timestamp, &entry)
-				assert.Equal(t, testData.expectedTimestamps[i], timestamp, "entry: %d", i)
+				out := processEntries(s, Entry{
+					Extracted: extracted,
+					Entry: api.Entry{
+						Labels: inputEntry.labels,
+						Entry: logproto.Entry{
+							Line:      entry,
+							Timestamp: timestamp,
+						},
+					},
+				})[0]
```

Comment:
The `Entry` construction spans all tests. Would it make sense to add a `NewEntry` method? This could ease groking the code.